### DISCUSSION TITLE
Backup settings and dock space at the bottom

### DISCUSSION THREAD

**Discussion by RainGater** [MonwF/customiuizer#199] [Q&A] [answered]
Hi,

First of all, I have to admit that CustoMIUIzer14 is an amazing module and makes me NOT miss AOSP one bit. Kudos to the developers for bringing all the necessary features that should have been enabled by default, imo.

I have a couple of questions as I am using CustoMIUIzer14 on Xiaomi 11T Pro running MIUI 14.0.1 global build.

1. When I hit Backup to backup the settings, I get a toast saying that I have to enable permissions manually. When I go to the app info for CustoMIUIzer, I don't see any option to enable permission for storage. What permission is the app asking for so that I can backup the settings?

2. There is lots of empty space below the dock in the Launcher. How to remove it?  I tried the Bottom dock margin and it isn't doing anything.

**Comment by MonwF** [comment]
1. customiuizer is running as system app. so you can't grant storage permission manually. i will upgrade the storage  mechanism in next month.
2. you can't remove the gap with customiuizer.

**Reply by RainGater** [reply]
Thank you and appreciate the fast response.  Looking forward to saving the settings in the not so distant future.

Btw, can't you please provide a bottom grid margin just like the top grid margin?  If it is possible that'll be a great solution. If not, I completely understand. 

Thanks again for all the wonderful work on this amazing module.

**Comment by MonwF** [comment, marked as answer]
i added a temporary fix.
see the issue: https://github.com/MonwF/customiuizer/issues/200

**Reply by RainGater** [reply]
Backups are working fine and thank you so much.

And, for my other issue, the gap below the dock cannot be removed even with the bottom grid margin option, correct?
